Bottom: 6af7a0b398fb677207941ac76563e103b4897910
Top:    b0d67b49135a823ac6dab140f7002973eadacacd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-13 01:59:10 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
